README.md

```diff
@@ -139,7 +139,10 @@ If you'd like to build the Python bindings, you'll need to install a few
 additional dependencies.
 
 ```sh
-pip install  install -r ./llvm-project/mlir/python/requirements.txt
+pip install -r ./llvm-project/mlir/python/requirements.txt
+
+pip install pip install tensorflow-cpu # needed mainly for
+                                # stablehlo to tf-save-model connversion
```

Comment:
This should be a separate section. Maybe even in a readme in the `savedmodel` directory.

Reply:
Actually without this dependency the tests cannot be build when `STABLEHLO_ENABLE_BINDINGS_PYTHON` is ON. Added a clarification note in the curent readme file. PTAL.


Also, added a `README.md` at `stablehlo.integrations/python/savedmodel` to provide clarification on the dependency.

Reply:
That's a merge-blocker IMO... we shouldn't (/ can't) require TF to build StableHLO.

Is it a build issue or test issue? We shouldn't require TF dependency for build. For testing we can define a separate test group (like `check-stablehlo-quick`) which requires the dependency.

Reply:
This is just a test issue. Created a separate section in the top level README about StableHLO TF integration and used stablehlo/savedmodel/README.md to explain the dependencies and testing instructions. PTAL.